File: Software Technologies/Exams/RetakeExam-5Sep2017/C#/AnimeList/Models/Anime.cs
using System.ComponentModel.DataAnnotations;
using System.Web.Mvc;

namespace AnimeList.Models
{
    public class Anime
    {
        [Key]
        public int Id { get; set; }

        [Required]
        public int Rating { get; set; }

        [Required]
        [AllowHtml]
        [StringLength(50)]
        public string Name { get; set; }

        [Required]
        [AllowHtml]
        public string Description { get; set; }

        [Required]
        [AllowHtml]
        public string Watched { get; set; }

    }
}

File: Software Technologies/Exams/RetakeExam-5Sep2017/JavaScript/models/Anime.js
const mongoose = require('mongoose');

let animeSchema = mongoose.Schema({
    rating: {type: Number, required: true},
    name: {type: String, required: true},
    description: {type: String, required: true},
    watched: {type: String, required: true}
});

let Anime = mongoose.model('Anime', animeSchema);

module.exports = Anime;